src/test/transaction_tests.cpp

```diff
@@ -375,6 +375,18 @@ BOOST_AUTO_TEST_CASE(tx_no_inputs)
     BOOST_CHECK(state.GetRejectReason() == "bad-txns-vin-empty");
 }
 
+BOOST_AUTO_TEST_CASE(tx_oversized)
+{
+    CMutableTransaction tx;
+    tx.vin.resize(1);
+    auto largeOutput = CScript() << OP_RETURN << std::vector<unsigned char>(MAX_BLOCK_WEIGHT / WITNESS_SCALE_FACTOR, 0x00);
```

Comment:
In 0aa7db42564408edb41b0d42103d39ba4c2787dc "Validate oversized transaction"

I think it would be more interesting for this test to be on the threshold exactly. The rest of the tx is 70 bytes, so `MAX_BLOCK_WEIGHT / WITNESS_SCALE_FACTOR - 70` should pass `CheckTransaction`, while `- 69` does not.

Reply:
I wanted a value that's definitely oversized, to avoid being close to the threshold - but I think I managed to make it work, thanks for the hint, let me know if you'd like me to reduce the duplication in what I've pushed.